### PR TITLE
Upgraded the Images for Workflow executor and controller.

### PR DESCRIPTION
**Pull request by Jonsy13**
**What this PR does / why we need it**:

This PR will update the images for controller and executor.
Also, will add the multiarch support for 
- k8s:latest
- litmus-checker
- litmus-git-app-checker
- app-qps-test
- app-deployer
- app-checker

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: custom/workflow-helper/app-checker/buildscripts/push [hunk 1]
```diff
@@ -8,7 +8,7 @@ then
   docker login -u "${DNAME}" -p "${DPASS}";
   #Push to docker hub repository with latest tag
   docker tag ${IMAGEID} litmuschaos/litmus-git-app-checker:latest
-  docker push litmuschaos/litmus-git-app-checker:latest;
+   docker buildx build . -f Dockerfile --progress plane --push --no-cache --platform linux/amd64,linux/arm64 --tag litmuschaos/litmus-git-app-checker:latest;
```
Comment (uditgaurav) [inline custom/workflow-helper/app-checker/buildscripts/push:11]: We don't need `docker tag` command and `IMAGEID` (line number 4)

File: custom/workflow-helper/app-deployer/Dockerfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ FROM golang:1.14 AS builder
 LABEL maintainer="LitmusChaos"
 
 ARG TARGETOS=linux
-ARG TARGETARCH=amd64
+ARG TARGETARCH
```
Comment (uditgaurav) [inline custom/workflow-helper/app-deployer/Dockerfile:7]: Can we do this for all multi-arch images?

File: custom/k8s/buildscripts/push [hunk 3]
```diff
@@ -8,7 +8,7 @@ then
   docker login -u "${DNAME}" -p "${DPASS}";
   #Push to docker hub repository with latest tag
   docker tag ${IMAGEID} litmuschaos/k8s:latest
-  docker push litmuschaos/k8s:latest;
+  docker buildx build -f Dockerfile --progress plane --push --no-cache --platform linux/amd64,linux/arm64 --tag litmuschaos/k8s:latest
```
Comment (uditgaurav) [inline custom/k8s/buildscripts/push:11]: We don't need `docker tag` command and `IMAGEID` (line number 4)

File: custom/k8s/Dockerfile [hunk 4]
```diff
@@ -1 +1,16 @@
-FROM alpine/k8s:1.18.2
\ No newline at end of file
+FROM alpine:3.12
+ARG TARGETPLATFORM
+
+LABEL maintainer="LitmusChaos"
+
+#Installing
```
Comment (uditgaurav) [inline custom/k8s/Dockerfile:6]: can we check this comment

File: .github/workflows/push.yml [hunk 5]
```diff
@@ -185,17 +185,35 @@ jobs:
...
-      - name: Build and Push litmus-git-app-checker image
-        run: make litmus-git-app-checker
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+
+      - name: Build and Push litmus-checker image
```
Comment (oumkale) [inline .github/workflows/push.yml:195]: Can we change this to ```Build and Push litmus-git-app-checker image```

File: .github/workflows/push.yml [hunk 6]
```diff
@@ -185,17 +185,35 @@ jobs:
...
+        run: |
+          make docker.buildx
+          make _push_litmus_api_checker
```
Comment (oumkale) [inline .github/workflows/push.yml:198]: make _push_litmus_git_app_checker

File: .github/workflows/push.yml [hunk 7]
```diff
@@ -185,17 +185,35 @@ jobs:
...
-      - name: Build and Push litmus-k8s image
-        run: make litmus-k8s
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+
+      - name: Build and Push litmus-checker image
```
Comment (oumkale) [inline .github/workflows/push.yml:213]: Same here ```litmus-k8s```

**Comment by Jonsy13** [comment]
Requested changes have been added. PTAL @uditgaurav @oumkale

**Review by oumkale — DISMISSED** [review]

File: custom/k8s/Dockerfile [hunk 8]
```diff
@@ -1 +1,14 @@
...
+LABEL maintainer="LitmusChaos"
+
+#Installing minimal packages
+RUN apk update && \
+    apk upgrade --update-cache --available 
+RUN apk --no-cache add curl &&\
+    rm -rf /var/cache/apk/*
+
+#Installing kubectl
+RUN curl -sLO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/${TARGETARCH}/kubectl" && chmod +x kubectl && mv kubectl /usr/bin/kubectl
```
Comment (uditgaurav) [inline custom/k8s/Dockerfile:13]:
```suggestion
ENV KUBECTL_VERSION="v1.19.0"
RUN curl -sLO "https://dl.k8s.io/release/${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl" && chmod +x kubectl && mv kubectl /usr/bin/kubectl
```
can we use specific version in place of latest version?

**Review by uditgaurav — APPROVED** [review]

**Review by oumkale — APPROVED** [review]

**Pull request merged**